otelcol/collector_windows.go

```diff
@@ -202,7 +202,7 @@ func (w windowsEventLogCore) Sync() error {
 
 func withWindowsCore(elog *eventlog.Log, serviceConfig **service.Config) func(zapcore.Core) zapcore.Core {
 	return func(core zapcore.Core) zapcore.Core {
-		if serviceConfig != nil {
+		if serviceConfig != nil && *serviceConfig != nil {
```

Comment:
@mx-psi fixed the windows test.  The issue was that when the config could not resolve the underlying serviceConfig was nil, but this check was only checking the pointer to the pointer not being nil.  When we passed the options to the fallback logger we ended up in this method and tried to do `(*serviceConfig).Telemetry.Logs.OutputPaths` on `nil`.  Added an extra nil check to make sure the underlying `serviceConfig` is also not nil.

Reply:
Thanks for getting to the bottom of this Windows test, can we make you the official Windows reviewer? 😄

Reply:
No thank you